utils/spiel/m0_filesystem_stats

```diff
@@ -150,12 +150,25 @@ if [ $rc -ne 0 ] ; then
 fi
 }
 
+get_hostname()
+{
+        host=" "
+        if [[ -f /etc/salt/minion_id ]] ; then
+            host=$(cat /etc/salt/minion_id)
+        fi
+
+        if [ -z $host ] ; then
```

Comment:
[defect]. Please write as this:
```
if [ -z "$host" ] ; then
```

String must be quoted, otherwise this condition will always be true.

Reply:
This should not affect for zero check, it is recommended to use "" for string comparision.
have did this changes.